docs/introduction/installation.md

```diff
@@ -0,0 +1,144 @@
+---
+sidebar_position: 2
+tags:
+  - Installation
+  - Getting started
+  - Quick start
+  - Docker
+  - Unraid
+  - Docker-Compose
+  - Source
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# 🚀 Installation
+
+Homarr can be installed trough 3 different methods. For most users, [the installation using Docker](#-installation-using-docker) will be sufficient and the easiest.
+
+## 🐋 Installation using Docker
+
+Docker is a free Software that isolates Homarr using containers. Using Docker, you can run Homarr on any supported plattform (including Windows and Raspberry Pi), without running any installers or linux knowledge.
+
+### Initial Installation
+
+To add the Homarr container via Docker, simply run the following command in your terminal.
+
+```bash
+docker run  \
+  --name homarr \
+  --restart unless-stopped \
+  -p 7575:7575 \
+  -v ./homarr/configs:/app/data/configs \
+  -v ./homarr/icons:/app/public/icons \
```

Comment:
This command won't work. Docker run needs absolute paths (I've had reports of it but didn't test)

Reply:
I quickly tried on my new Raspberry Pi. You're right, relative paths are not support. TIL 😅